docs/pages/installation.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 'Installation guide'
+weight: 30
+---
+
+## Glee Installation
+
+Before installing Glee into your project, make sure you have pre-installed NPM, NodeJs and [AsyncAPI CLI](https://github.com/asyncapi/cli) tools on your system.
+
+The best way to get started with Glee is by using AsyncAPI CLI, which sets up everything automatically for you. 
+To create a project, run:
+
+```sh
+asyncapi new glee
+```
+
+> For more information on how to install the AsynAPI CLI, you can review the [CLI installation guide](https://www.asyncapi.com/docs/tools/cli/installation).
+
+On installation, you'll find next steps after your project created:
+
+```
+Your project "project" has been created successfully!
+
+Next steps:
+
+  cd project
+  npm install
+  npm run dev
+
+Also, you can already open the project in your favorite editor and start tweaking it
+```
+
+While making twists to your application, you can follow along with our getting started guide on the relevant page.
```

Comment:
I think adding the link will be useful.

Reply:
Yes! I will do that once page rendered or reviewed with a link